frontend/app/views/fragments/form/billingAddressPayPal.scala.html

```diff
@@ -0,0 +1,33 @@
+@(heading: String)
+
+@pseudoRadio(title: String) = {
+    <div class="pseudo-radio">
+        <span class="pseudo-radio__label">@title</span>
+    </div>
+}
+
+<fieldset class="fieldset js-toggle-billing-address">
+
+    <div class="fieldset__heading">
+        <h2 class="fieldset__headline">@heading</h2>
+    </div>
+
+    <div class="fieldset__fields">
+
+        <div class="form-field billing-choice js-billing-input-container">
+
+            <label class="label" for="use-delivery-address">
+                <input class="is-hidden" type="radio" name="address.type" id="use-delivery-address" value="true" checked="checked"/>
+                @pseudoRadio("Use delivery address")
+            </label>
+
+            <label class="label" for="use-billing-address">
+                <input class="is-hidden js-use-billing-address" type="radio" name="address.type" id="use-billing-address" value="false"/>
```

Comment:
What's the deal with `js-use-billing-address`? Doesn't appear to be defined anywhere?